dependencies/pom.xml

```diff
@@ -686,6 +688,21 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
```

Comment:
These dependencies should not be defined in this file.

Reply:
Done